### PR TITLE
Bug 1475440 Corrected vxlan naming info

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1475440

@sferich888 as discussed in the BZ. Please let me know if there's anything more needed for this BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
Verified with SMEs over email that what's there should be good. @openshift/team-documentation PTAL

File: admin_guide/sdn_troubleshooting.adoc [hunk 1]
```diff
@@ -360,9 +360,8 @@ on this bridge.
 cluster subnet gateway address, and is used for external network
 access. OpenShift SDN configures `netfilter` and routing rules to enable access
 from the cluster subnet to the external network via NAT.
-* `vxlan0`: The OVS VXLAN device (port 1 on `br0`), which provides access to
-containers on remote nodes.
-* `vethX` (in the main netns): A Linux virtual ethernet peer of `eth0` in the docker netns.  It will be attached to the OVS bridge on one of the other ports.
+* `vxlan_sys_4789`: The OVS VXLAN device (port 1 on `br0`), which provides access to containers on remote nodes. Referred to as `vxlan0` in the OVS rules.
+* `vethX` (in the main netns): A Linux virtual ethernet peer of `eth0` in the docker netns. It will be attached to the OVS bridge on one of the other ports.
```
Comment (mburke5678) [inline admin_guide/sdn_troubleshooting.adoc]: is "docker netns" properly formatted? *docker* netns ??
Should "docker" be "Docker"?

**Comment by mburke5678** [comment]
@bfallonf Couple of nits.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../admin_guide/sdn_troubleshooting.adoc#admin-guide-sdn-troubleshooting[Troubleshooting OpenShift SDN]
|Corrected `vxlan0` to `vxlan_sys_4789` in the xref:../admin_guide/sdn_troubleshooting.adoc#debugging-local-networking[Debugging Local Networking] section.
%
|xref:../architecture/additional_concepts/sdn.adoc#architecture-additional-concepts-sdn[Additional Concepts -> OpenShift SDN]
|Corrected `vxlan0` to `vxlan_sys_4789` in the xref:../architecture/additional_concepts/sdn.adoc#sdn-design-on-nodes[Design on Nodes] section.
%

**Comment by bfallonf** [comment]
Thanks @mburke5678 . I corrected, and merged.

**Pull request merged**